editor/debugger/editor_debugger_node.cpp

```diff
@@ -88,7 +97,20 @@ ScriptEditorDebugger *EditorDebuggerNode::_add_debugger(String p_name) {
 	node->connect_compat("remote_object_updated", this, "_remote_object_updated", varray(id));
 	node->connect_compat("remote_object_property_updated", this, "_remote_object_property_updated", varray(id));
 	node->connect_compat("remote_object_requested", this, "_remote_object_requested", varray(id));
-	add_child(node);
+
+	if (tabs->get_tab_count() > 0) {
+		Object::cast_to<ScriptEditorDebugger>(tabs->get_tab_control(0))->clear_style();
```

Comment:
`get_debugger(0)->clear_style();`

But even better, since now the `EditorDebuggerNode` handle the `"panel"` style override of child sessions, maybe it should be removed from `ScriptEditorDebugger` completely.
For instance, right now changing the editor settings will result in the style of the child sessions to be updated, reverting the changes of this `clear_styles` call.

Reply:
You mean overriding `ScriptEditorDebugger`'s style directly from `EditorDebuggerNode`?

Reply:
Actually, I think it's fine like this for now.